tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:
+    from unittest.mock import patch, call
+except ImportError:
+    from mock import patch, call
+
+
+class TestPasswordFile(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestPasswordFile, self).__init__(*args, **kwargs)
+        self.config_file = None
+        self.pwd_file = None
+        self.cwd = os.getcwd()
+        self.tabpy_cwd = os.path.join(self.cwd, 'tabpy-server', 'tabpy_server')
+
+    def setUp(self):
+        os.chdir(self.tabpy_cwd)
+        self.config_file = NamedTemporaryFile(mode='w', delete=False)

```

Comment:
What is the point of creating these files to subsequently close?

Reply:
The files need to be closed for their content to be available for a reader. The options were either I do it here and then in`_set_file` open the file for writing, or keep it open and close in `_set_file`